src/Uno.UI.XamlHost.Skia.Wpf/UnoXamlHost.cs

```diff
@@ -14,9 +15,29 @@ namespace Uno.UI.XamlHost.Skia.Wpf
 	/// </summary>
 	public partial class UnoXamlHost : UnoXamlHostBase
 	{
+		private static readonly Style _style = (Style)XamlReader.Parse(
```

Comment:
The `XamlReader.Parse` will cause the startup perform issues. Can we put the style with `x:Key` to ResourceDictionary.xaml and reference it by Key here?

Reply:
Unfortunately not. The problem is that the xaml files are no longer read due to the recent structural changes.